### PR TITLE
Change title tag for news items

### PR DESCRIPTION
**Pull request by shjohnson**
### Jira link

HOTT-3814

### What?

I have added/removed/altered:

- [ ] Change title tag format for news items

### CODE REVIEW

File: app/views/news_items/show.html.erb [hunk 1]
```diff
@@ -1,4 +1,5 @@
-<%= content_for :head, news_feed_link_tag %>
+<% content_for :head, news_feed_link_tag %>
+<% content_for :title, "#{@news_item.title} - #{t('news_item.title.'+"#{@news_item.service_name}")}" %>
```
Comment (willfish) [inline app/views/news_items/show.html.erb:2]: I think there might be a helper for this

Comment (shjohnson) [reply app/views/news_items/show.html.erb:2]: Which part? I should have added the content for head i dont think was working properly before

Comment (shjohnson) [reply app/views/news_items/show.html.erb:2]: I tried looking for any helpers but this is closest i could fine.

Comment (willfish) [reply app/views/news_items/show.html.erb:2]: Okey poke. I had a look and couldn't see a generic version for this

**Review by willfish — APPROVED** [review]

**Pull request merged**